automatic/sysinternals/tools/chocolateyInstall.ps1

```diff
@@ -3,32 +3,46 @@ $packageName = 'sysinternals'
 $url = 'https://live.sysinternals.com/files/SysinternalsSuite.zip'
 $installDir = Split-Path -parent $MyInvocation.MyCommand.Definition
 
-Install-ChocolateyZipPackage $packageName $url $installDir
+# Default the values
+$defaultInstallDir = Split-Path -parent $MyInvocation.MyCommand.Definition
+$installDir = $defaultInstallDir
+
+# Now parse the packageParameters using good old regular expression
+if ($packageParameters -and $packageParameters.Length -gt 18) {

```

Comment:
For the installationpath...

Why not call it something shorter? Like `/installdirectory`?